### PR TITLE
Remove icon-shim dependency

### PR DESCRIPTION
**Pull request by jglick**
Not sure why it was here—was actually older than what you would get by default anyway. Cf. https://github.com/jenkinsci/plugin-pom/pull/72.

@reviewbybees

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -33,6 +33,7 @@
   <properties>
     <jenkins.version>1.642.4</jenkins.version>
     <java.level>7</java.level>
+    <jenkins-test-harness.version>2.30-20171010.153224-1</jenkins-test-harness.version> <!-- TODO https://github.com/jenkinsci/jenkins-test-harness/pull/78 -->
```
Comment (oleg-nenashev) [inline pom.xml]: I would propose to release JTH and Plugin POM before merging this PR

Comment (jglick) [reply pom.xml]: yes, would do

Comment (jglick) [reply pom.xml]: Well, JTH at least. `plugin-pom` can come along later, not urgent.

**Comment by KostyaSha** [comment]
> Not sure why it was here

Because of docker-traceability. Lol, seems I remember introduced docker chaos better.

**Comment by KostyaSha** [comment]
So now you broke dependent plugins??
I guess identity was needed for  https://github.com/jenkinsci/docker-traceability-plugin/blob/master/docker-traceability-plugin/pom.xml#L141 
And icon shim for docker icons.

**Comment by jglick** [comment]
It should still get `icon-shim` from core. A slightly later version in fact.

**Comment by KostyaSha** [comment]
@jglick Then it looks safe.  Afair this plugin was created as piece of traceability and later reused in other plugins.

**Pull request merged**